Source/CarthageKit/Project.swift

```diff
@@ -120,6 +120,9 @@ public enum ProjectEvent {
 	/// Building the project is being skipped, since the project is not sharing
 	/// any framework schemes.
 	case skippedBuilding(ProjectIdentifier, String)
+	
+	/// Building the project is being skipped because it is cached.
+	case skippedBuildingCached(ProjectIdentifier)
```

Comment:
@mdiep Added logging to report when a valid/invalid cache or no cache is found.  I believe this is PR ready for another review.